mm2src/mm2_main/tests/docker_tests/eth_docker_tests.rs

```diff
@@ -357,32 +402,74 @@ pub enum TestNftType {
     Erc721 { token_id: u32 },
 }
 
+pub(crate) fn init_p2p_context(ctx: &MmArc) {
```

Comment:
I don't understand why this is needed? can you please clarify?

Reply:
Sorry! I thought it was reverted in "Revert" commit, but seems like other "Revert" commit added it again.

We dont need it https://github.com/KomodoPlatform/komodo-defi-framework/pull/2169/commits/653c063bce33af9b0af4e85e3943a088e192f8f0